ocs_ci/deployment/helpers/hypershift_base.py

```
@@ -1161,22 +1159,42 @@ def download_hosted_cluster_kubeconfig(
+                os.path.isfile(kubeconfig_path)
+                and os.stat(kubeconfig_path).st_size == 0
+            ):
+                os.remove(kubeconfig_path)
             return
 
-        if not os.stat(kubeconfig_path).st_size > 0:
+        if not (
+            os.path.isfile(kubeconfig_path) and os.stat(kubeconfig_path).st_size > 0
+        ):
             logger.error(
                 f"Failed to download kubeconfig for HyperShift hosted cluster {name}"
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)